core/config/locales/en.yml

```diff
@@ -1282,7 +1282,7 @@ en:
     inventory: Inventory
     inventory_adjustment: Inventory Adjustment
     inventory_canceled: Inventory canceled
-    inventory_error_flash_for_insufficient_quantity: An item in your cart has become unavailable.
+    inventory_error_flash_for_insufficient_quantity: "'#{names}' has become unavailable."
```

Comment:
Valid points. I think we could just remove the apostrophe from the translation. Stores that want to keep it, can easily update their locale file. And "has become" could be "became" and everything should be fine, no?

WDYT @jhawthorn @ericsaupe